index.Rmd

```diff
@@ -1,17 +1,17 @@
 ---
 title: "Opoid-Related Deaths in New York State"
 author: "Anuwat Pengput"
-subtitle: Spatial Epidemiological Analysis and Risk Factors of Opoid-Related Deathsin
+subtitle: Spatial Epidemiological Analysis and Risk Factors of Opoid-Related Deaths in
   New York
 output:
   html_document:
     df_print: paged
 ---
 
-# Introduction
+# Introduction 
```

Comment:
There's a lot of information here that lays the foundation for the analysis to come.  It could be improved by honing in on a central question to be answered.